src/components/DatePicker.tsx

```diff
@@ -11,6 +10,8 @@ import Button from "./Button";
 import TextField, { Props as TextFieldProps } from "./TextField";
 import Touchable from "./Touchable";
 import theme from "../styles/DefaultTheme";
+//@ts-ignore
+import DateTimePicker from "./DatePickerComponent";
```

Comment:
you can call this DateTimePicker.native.tsx and DateTimePicker.web.tsx

Reply:
renamed to DateTimePicker.tsx and DateTimePicker.web.tsx. No ts errors, native picks up normal one and web picks up web